op-challenger/flags/flags_test.go

```diff
@@ -40,7 +40,7 @@ func TestCorrectEnvVarPrefix(t *testing.T) {
 		if envVar == "" {
 			t.Errorf("Failed to find EnvVar for flag %v", flag.GetName())
 		}
-		if envVar[:len("OP_CHALLENGER_")] != "OP_CHALLENGER_" {
+		if !strings.HasPrefix("OP_CHALLENGER_", envVar) {
```

Comment:
I think the args need to go the other way around.

Reply:
yep